schema.graphql

```diff
@@ -125,8 +167,13 @@ type ProductionNode implements Node {
   coverImage: GrapheneImageFieldNode
   ageRating: Int
   facebookEvent: String
+  warnings(offset: Int, before: String, after: String, first: Int, last: Int): WarningNodeConnection!
   slug: String!
+  cast(offset: Int, before: String, after: String, first: Int, last: Int): CastMemberNodeConnection!
+  crew(offset: Int, before: String, after: String, first: Int, last: Int): CrewMemberNodeConnection!
```

Comment:
Tempted to say same as above for cast crew and prod team - it's not worth paginating them because of how we display